libs/content/workspace/src/workspace.rs

```diff
@@ -382,6 +430,10 @@ impl Workspace {
     fn show_tabs(&mut self, ui: &mut egui::Ui) {
         ui.spacing_mut().item_spacing = egui::vec2(0.0, 0.0);
 
+        if self.active_tab_changed {
+            self.cfg.update_last_open_tabs(&self.tabs, self.active_tab);
+        }
```

Comment:
Only update the persisted tabs when the active tab changes? Does closing an inactive tab correctly update the persisted tabs? I can also see us implementing features that would open a file without setting it active (think cmd+clicking a link in chrome)

Reply:
update: closing an inactive tab does work (I'm not sure how)